.github/workflows/reusable-prober.yml

```diff
@@ -17,11 +17,6 @@ on:
         type: string
         default: 'https://fulcio.sigstore.dev'
         description: 'Fulcio URL'
-      fulcio_grpc_url:
-        required: false
-        type: string
-        default: 'fulcio.sigstore.dev'
-        description: 'Fulcio GRPC URL'
       tsa_url:
         required: false
         type: string
```

Comment:
We have a trusted root as input already, but we'll also want one for signing config.